commcare_connect/templates/opportunity/opportunity_list.html

```diff
@@ -15,19 +72,23 @@ <h1> Opportunities
         </span>
       </h1>
     </div>
-    <div class="pb-4">
+    <div class="pb-4" x-data="sortableTable()">
       <table class="table border table-responsive">
         <thead class="table-light">
         <tr>
-          <th>Name</th>
+          <th @click="sortByColumn" role="button">Name <i class="bi bi-arrow-down-short"></i></th>
+          <th @click="sortByColumn" role="button">Start Date <i class="bi bi-arrow-down-short"></i></th>
```

Comment:
Does the arrow switch direction when the columns are sorted in the opposite order?

Reply:
Not really, apparently that requires good amount of Javascript and CSS. Best handled by libraries such as datatable